invenio_app_rdm/records_ui/previewer/mirador.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2022 data-futures.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Mirador preview."""
+
+from os.path import splitext
+
+from flask import current_app, render_template
+from werkzeug.local import LocalProxy
+
+previewable_extensions = LocalProxy(lambda: current_app.config["IIIF_FORMATS"].keys())
```

Comment:
with this configuration, we cannot choose anymore if a JPEG will be rendered with Mirador or as a simple image. All images will be rendered with Mirador.
Is this what we want?

Reply:
Agree, we will split into a separate `MIRADOR_PREVIEW_EXTENSIONS` config